File Upload Dropzone Accessibility Fixes

coliff opened the pull request:
## Description

- Link should be underlined - **FIXED**
- Error Message should have `role="alert"`   (REF: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role) - **FIXED**
- Make the 'Drag files here or browse to upload' a single string to make customizing it in the future easier. - **FIXED**
- Update the SVG icon to use https://modus-icons.trimble.com/modus-solid/cloud-upload/ - **FIXED**

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## How Has This Been Tested?

Edge v117

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

msankaran0712 reviewed (changes requested):
Something minor, the icon does not look to be centered 
![image](https://github.com/trimble-oss/modus-web-components/assets/50131391/e1937d6c-7ae1-49ae-8a5a-1acf5109b21c)

coliff commented:
thanks for spotting that. That was an error on my part, the updated SVG I added should have viewport 24 x 24, not 32x32 as it was before. Fixed now locally and committing
